lib/compiler_rt/memcpy.zig

```diff
@@ -43,11 +43,13 @@ pub fn memcpy(noalias dest: ?[*]u8, noalias src: ?[*]const u8, len: usize) callc
         return dest;
     }
 
-    inline for (5..std.math.log2(2 * size) + 1) |p| {
-        const limit = 1 << p;
-        if (len <= limit) {
-            memcpy_range2(limit / 2, dest.?, src.?, len);
-            return dest;
+    if (5 <= std.math.log2(2 * size)) {
```

Comment:
Completely unncessary comment but perhaps you can simplify this to 16 <= size now. :)

Newbie zig question: is this log2 comptime evaluated anyway?

Reply:
That log2 in the if statement wouldn't be evaluated at comptime (and it should be) - it should have a `comptime` keyword added. llvm was inlining/constant folding it so there wasn't actually a runtime test when using the llvm backend.

While `16 <= size` is simpler in isolation (and more so when the `comptime` is added, I think changing the test to compare to 16 makes it less clear why the test is needed.